### DISCUSSION TITLE
Nozzle tips

### DISCUSSION THREAD

**Discussion by bonz099** [xpDIY/3DPlacer#8] [General]
Did you buy your tips from a vendor, or print them? If bought, would you mind posting a link to them?

**Comment by xpDIY** [comment]
Nozzle tip is from 3rd party, or if you have a resin printer that should also work for you, the typical vender could be PCBWays or JLCPCB, they both offer 3D printing service, and some vender will warn about the too thin wall, but that is ok, just let them continue. Remember to choose black resin, otherwise you got problem of camera recognition.

**Comment by bonz099** [comment]
Thank you, could you provide me with a value I could search for? Ide like to see if I can just buy them from a vendor instead of getting printed ones.

**Reply by xpDIY** [reply]
so maybe a bit misunderstanding, the vender I mean is the one who providing 3D printing service, so I took the model and ask them to print out, and ship to me, so it is actually 3D printed, I didn't mean buying directly from some vender.

**Reply by bonz099** [reply]
Ok, i got it. I think I'm going to make an insert to use these cheap and easily available PNP tips. 

https://www.amazon.com/gp/product/B09JBWYYST/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1
